schema/360-giving-package-schema.json

```diff
@@ -17,6 +17,15 @@
           "type": "string",
           "pattern": "^(\\d+\\.)(\\d+)$"
         },
+	"extensions": {
+	  "title": "Extensions",
+	  "description": "An array of strings which refer to Extensions in use by this 360Giving data file. Please see the Guidance for details.",
+	  "type": "array",
+	  "items": {
+	    "type": "string"
+	  },
+	  "uniqueItems": true
```

Comment:
As per note, I think forcing the extension codes to be unique is good practise and while not explicitly included in the Extensions proposal, is in keeping with the goals of this update. I'm happy to approve the update with this feature included.